client/src/main/resources/swaggercodegen/generatedAnnotation.mustache

```diff
@@ -0,0 +1,13 @@
+{{!
+  Copied from the https://github.com/swagger-api/swagger-codegen-generators tagged release 1.0.24:
```

Comment:
Any reason to use 1.0.24 vs the latest 1.0.42? apples to other files as well

Reply:
The swagger code gen dependency version is the same (1 patch behind) as WSM. Didn't want to risk having any side effects from bumping both.